helm-info.el

```diff
@@ -240,15 +241,71 @@ helm-info-<CANDIDATE>."
       (funcall helm-info-function)
       (ring-insert helm-info-searched candidate))))
 
+(defvar helm-info--files-cache nil)
+(defvar helm-info--files-doc-cache (make-hash-table :test 'equal))
+
+(defun helm-info-file-doc (file)
+  "Return dir entry from the Info FILE."
+  (or
+   (with-temp-buffer
+     (ignore-errors (info-insert-file-contents file))
+     (goto-char (point-min))
+     (when (re-search-forward (rx line-start "START-INFO-DIR-ENTRY" line-end)
+                              nil t)
```

Comment:
Please use plain regexps, not RX, I don't understand it, it's easier for me to read plain regexps.